pkg/security/probe/probe_ebpf.go

```diff
@@ -1628,6 +1628,11 @@ func (p *EBPFProbe) DumpProcessCache(withArgs bool) (string, error) {
 	return p.Resolvers.ProcessResolver.ToDot(withArgs)
 }
 
+// EnableEnforcement sets the enforcement mode
+func (p *EBPFProbe) EnableEnforcement(state bool) {
+	p.processKiller.enabled = state
```

Comment:
Shouldn't this use `processKiller.SetState(state)` instead?

Reply:
You're right. I forgot I added this helper :-)